app/views/claims/support/schools/index.html.erb

```diff
@@ -1,6 +1,7 @@
-<div class="govuk-grid-row">
-  <div class="govuk-grid-column-two-thirds">
-    <h1 class="govuk-heading-xl"><%= t("organisations") %></h1>
+<div class="govuk-width-container">
```

Comment:
I think the link to go to the show page of the support school in this file is wrong https://github.com/DFE-Digital/itt-mentor-services/blob/ca4b8ab05f17e099c6b681847ff15d3af2a9e846/app/views/claims/support/schools/index.html.erb#L13

It should go to support/school show page but it just goes to school/show page. This is my bad https://github.com/DFE-Digital/itt-mentor-services/pull/87, I can fix it in my next PR, I'm only saying this because it's hard to test this PR without fixing this issue

Reply:
I'll update 😄